sentry-resque/lib/sentry/resque.rb

```diff
@@ -32,6 +32,15 @@ def record(queue, worker, payload, &block)
 
               finish_transaction(transaction, 200)
             rescue Exception => exception
+              klass = payload['class'].constantize
+
+              if Sentry.configuration.resque.report_after_job_retries && defined?(::Resque::Plugins::Retry) == 'constant' && klass.is_a?(::Resque::Plugins::Retry)
+                retry_key = klass.redis_retry_key(payload['args'])
+                retry_attempt = ::Resque.redis.get(retry_key)
```

Comment:
the SDK should never do its own database calls by any chance, that's a side effect that breaks all sorts of contracts.
Is there no way of knowing the retry status from the stuff available to this function?

Reply:
@sl0thentr0py  In fact, there is. I didn't realize it, but in fact, we are catching an exception here (so the processed job context is available) and when we use constantize against the class name, resque-retry will capture the job context, so methods `retry_limit_reached?` and `retry_limit_attempt` will expose the current "status" of the job.

I just pushed a new commit that fixes it.